src/main/java/org/takes/rq/form/RqFormBase.java

```diff
@@ -76,7 +77,9 @@ public RqFormBase(final Request request) {
     public Iterable<String> param(final CharSequence key)
         throws IOException {
         final List<String> values = this.map().getOrDefault(
-            new EnglishLowerCase(key.toString()).string(),
+            new UncheckedText(
+                new Lowered(key.toString())
```

Comment:
@fanifieiev Isn't `key` already `String`? Why use `toString()` here?

Reply:
@fabriciofx Because key is CharSequence